packages/common/src/fixtures/preview-service/xforms/cities.geojson

```diff
@@ -0,0 +1,65 @@
+{
+    "type": "FeatureCollection",
+    "features": [
+        {
+            "type": "Feature",
+            "properties": {
+		"title": "Warsaw",
```

Comment:
Observation - no blocking: 

For some reason, the spacing for indentation displays differently in GitHub from properties like this `title` vs `info` Line 8. Are these different characters? 

Another one, L14 vs L15

Reply:
Looks like mixed tabs/spaces. FWIW I literally downloaded this file from Google Drive and committed as-is. As with any fixtures we've inherited from JR, I think it's a good idea to preserve the original formatting so we can clearly see substantive differences as it gets updated from upstream over time.